recipe/meta.yaml

```diff
@@ -21,7 +21,7 @@ requirements:
   host:
```

Comment:
We can change [this](https://github.com/AnacondaRecipes/yarl-feedstock/pull/5/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aL16) to `py<38` based on [this](https://github.com/aio-libs/yarl/blob/v1.11.0/setup.cfg#L56)

Reply:
The project was already configured with `skip: true  # [py<=37]`.

Reply:
I always thought it didn't recognize the `=` for skipping python variants, but maybe I'm mistaking that for `>`.